charts/gardener/controlplane/charts/runtime/templates/admission-controller/configmap-componentconfig.yaml

```diff
@@ -52,10 +52,14 @@ data:
         port: {{ required ".Values.global.admission.config.server.metrics.port is required" .Values.global.admission.config.server.metrics.port }}
       {{- if .Values.global.admission.config.server.resourceAdmissionConfiguration }}
       resourceAdmissionConfiguration:
+        {{- if .Values.global.admission.config.server.resourceAdmissionConfiguration.limits }}
```

Comment:
Nit: You could adapt the example in https://github.com/gardener/gardener/blob/bbc46e34f303c620081c5940dad20f21a23194a2/charts/gardener/controlplane/values.yaml#L310-L315